RAMP/module_metadata.py

```diff
@@ -4,6 +4,7 @@
 # Defaults
 ARCHITECTURE = platform.machine()
 OS = platform.system()
+OS_LIST = ["ubuntu14.04", "rhel7"]
```

Comment:
If this is a minimum version, i.e. ubuntu14.04 and above is supported, please add a comment mentioned that behavior.

Reply:
@swilly22 added it in the readme